src/transport/PairingSession.h

```diff
@@ -159,6 +160,11 @@ class DLL_EXPORT PairingSession
         return err;
     }
 
+    static CHIP_ERROR EncodeMRPParameters(TLV::Tag tag, const ReliableMessageProtocolConfig & mrpConfig,
+                                          System::PacketBufferTLVWriter & tlvWriter);
+
+    CHIP_ERROR DecodeMRPParametersIfPresent(System::PacketBufferTLVReader & tlvReader);
```

Comment:
Some documentation may help, to explain the expected format of the tlvReader. It seems this works only if tlvReader is positioned on a struct that contains MRP parameters, otherwise the MRP is assumed to be missing. Is this correct?

Can we unit test this? Code is self-contained enough that this seems possible and in that case we should do it.